Menu burger

SDecambray opened the pull request:
Add the header, the logo and the menu burger, we just need to stylish that ;)

Commit:
create a menuburger + add logo

Commit:
add navbar solid to transparent

LuigiMenez reviewed (dismissed):
Ok for me.

LoicBrassart reviewed (dismissed):
Saw no Evil

Dash293 committed:
Merge branch 'dev' into menu_burger

LuigiMenez commented on frontend/src/components/Header/style.jsx:
```diff
@@ -1,5 +1,147 @@
+  .navbar_logo {
+    justify-self: center;
+  }
+  .navbar_links {
```
First time

LuigiMenez replied:
First time

LuigiMenez commented on frontend/src/components/Header/style.jsx:
```diff
@@ -1,5 +1,147 @@
+  .navbar_links {
+    display: flex;
+  }
+  .navbar_link {
```
second time

LuigiMenez replied:
First time

LuigiMenez replied:
First time

LuigiMenez commented on frontend/src/components/Header/style.jsx:
```diff
@@ -1,5 +1,147 @@
+    border: none;
+    color: inherit;
+  }
+  .navbar_links {
```
second time

LuigiMenez commented on frontend/src/components/Header/style.jsx:
```diff
@@ -1,5 +1,147 @@
+    height: 1px;
+    background: #fff;
+  }
+  .navbar_link {
```
second time

LuigiMenez reviewed (changes requested):
It is better to use only one writing standard for the class name, preferred the camelCase

Dash293 committed:
syntax correction

we have rewritten some syntax errors like class names do not using camelCase

Dash293 committed:
class content and syntax correction

We have rewritten some syntax errors like class names do not using camelCase and removed double class selectors.

Dash293 committed:
Update style.jsx

LuigiMenez reviewed (approved):
Ok

LoicBrassart reviewed (approved):
Saw no Evil
